platform/error/platform-error-page-injector/src/main/java/org/codice/ddf/platform/error/injector/ErrorPageInjector.java

```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.platform.error.injector;
+
+import java.lang.reflect.Field;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import javax.servlet.ServletContext;
+import org.codice.ddf.platform.error.servlet.ErrorServlet;
+import org.eclipse.jetty.servlet.ErrorPageErrorHandler;
+import org.eclipse.jetty.servlet.ServletContextHandler;
+import org.eclipse.jetty.servlet.ServletHandler;
+import org.eclipse.jetty.servlet.ServletHolder;
+import org.osgi.framework.Bundle;
+import org.osgi.framework.BundleContext;
+import org.osgi.framework.ServiceEvent;
+import org.osgi.framework.hooks.service.EventListenerHook;
+import org.osgi.framework.hooks.service.ListenerHook.ListenerInfo;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ErrorPageInjector implements EventListenerHook {
+
+  private Map<String, String> errorCodesMap = new HashMap<>();
+  private final String errorPagePath = "/ErrorServlet";
+
+  public ErrorPageInjector() {
+    errorCodesMap.put("400", errorPagePath);
+    errorCodesMap.put("401", errorPagePath);
+    errorCodesMap.put("403", errorPagePath);
+    errorCodesMap.put("404", errorPagePath);
+    errorCodesMap.put("405", errorPagePath);
+    errorCodesMap.put("406", errorPagePath);
+    errorCodesMap.put("500", errorPagePath);
+    errorCodesMap.put("501", errorPagePath);
+    errorCodesMap.put("502", errorPagePath);
+    errorCodesMap.put("503", errorPagePath);
+    errorCodesMap.put("504", errorPagePath);
```

Comment:
✏️ You could look into Guava's way of initializing a map. You could just do something like `ImmutableMap.of()` and pass in all the String literals you need. And it wouldn't need to be in the constructor at that point. It could just be initialized on line 36. You should be able to find usage of that elsewhere in DDF.